src/script/ceph-backport.sh

```diff
@@ -232,6 +232,7 @@ function populate_original_pr {
 }
 
 function prepare {
+    local offset=0
```

Comment:
We could dispense with `offset` altogether - see my comment below.

Reply:
This is just the initialzation...`$offset` actually gets set immediately before the for loop. I just thought its good practice to set it to something deterministic.

Reply:
Yeah, I still think `offset` can be dropped, but I'm willing to live with it if I can get a more coherent debug message ;-)

Reply:
I'm a fan of keeping bash shenanigans like arithmetic with `(( ))` nice and explicit.  `git cherry-pick -x "pr-$original_pr~$(($i-1))"` is a bit eye watering imho. Debug message is updated...or should I add the commit offset there as well?